src/renderer/components/wallet/txs/send/SendFormBTC.tsx

```diff
@@ -20,8 +45,208 @@ type Props = {
   fees: FeesRD
 }
 
-const SendFormBTC: React.FC<Props> = (): JSX.Element => {
-  return <h1>SendFormBTC - coming soon ...</h1>
+const SendFormBTC: React.FC<Props> = (props: Props): JSX.Element => {
```

Comment:
lets dont use `(props: Props)`. at the `React.FC<Props>` this is a generic parameter which includes `children` prop at least

Reply:
Do you want `children` in `Props`? Or do you want to avoid it? _I'm not sure what you mean..._

In case of the former: `FunctionComponent` already provides it via `P` -> ` PropsWithChildren<P>`, so we don't miss the `children` prop here:

![Screenshot from 2020-09-30 10-39-26](https://user-images.githubusercontent.com/61792675/94662838-3d1f9100-0309-11eb-8539-63a7de1e7bc6.png)

Reply:
we will lose `children` props as we declared props as `Props` manually and type inferring will not work in this case.
just try to read `props.children` at this component and you will see what im talking about

Reply:
Resolved by a859e01e7d6a166f5e1a6dd24ea4cc43b577ec04